### ISSUE TITLE
sample gist - code

### ISSUE THREAD

**Issue by johndpope** [dmcyk/SwiftyCurl#3]
awesome job - 
It would be nice to see a before / after of ibm's curl vs this wrapper. 
Perhaps you could share a gist - and link in readme?

fyi - I wonder if you'd be interested in gluing this library with grpc services for server side swift?
https://github.com/grpc/grpc-swift/issues/2
https://github.com/nubbel/swift-tensorflow/issues/5

**Comment by dmcyk** [comment]
Hi, thanks! 
Here's a [gist](https://gist.github.com/dmcyk/5aceebcbe4b4117c9806c1cb01a8e205).
I've also updated readme. 

The project looks interesting, but I'm afraid I won't have enough time. I have loads of work at university, tomorrow I'm going home for Christmas break, I haven't seen my family for almost half year and once I'm back I'm instantly starting with university's finals :(. I may give it a try if I have enough time but I can't promise anything, sorry. 

If you encounter any issues with this project let me know, I will do my best to improve it, I've successfully used it in my internal projects but it's not really battle tested.
